apiserver/params/storage.go

```diff
@@ -730,3 +730,19 @@ type StorageAddParams struct {
 type StoragesAddParams struct {
 	Storages []StorageAddParams `json:"storages"`
 }
+
+// StorageAttachParams holds the parameters for a Storage.Attach API call.
+type StorageAttachParams struct {
+	Attachments []StorageAttachParam `json:"attachments,omitempty"`
+}
+
+// StorageAttachParam holds a parameter for a Storage.Attach API call.
+type StorageAttachParam struct {
```

Comment:
It's not *exactly* the same, but the difference is probably not worth a separate struct. StorageDetachParam has omitempty on the entity tag, StorageAttachParam does not.

Seeing as we're only accepting storage IDs now, I'll just change over to using the existing StorageAttachmentId(s) structs.